### ISSUE TITLE
Can not use flipIn open animation with fadeOut close animation

### ISSUE THREAD

**Issue by AAudusseau** [needim/noty#312]
There is my options : 

``` javascript
{
  text: 'My awesome text',
  type: 'error',
  layout: 'bottomLeft',
  timeout: false,
  animation: {
    open: 'animated flipInX',
    close: 'animated fadeOutUp'
  }
}
```

When I close the notification, the class _flipInX_ is not removed so the close animation (with class _fadeOutUp_) do not works.

Thanks for the help

**Comment by siddharthbhagwan** [comment]
#313

**Comment by needim** [comment]
fixed in v2.3.10
